client/client_test.go

```diff
@@ -62,8 +65,22 @@ func Test_Integ_NewEthereumPaymentClient(t *testing.T) {
 
 	t.Run("happy", func(t *testing.T) {
 		cfg.DatabaseDir = newDatabaseDir(t) // start with empty persistence dir each time.
-		_, err := client.NewEthereumPaymentClient(cfg, user, tcp.NewTCPBackend(5*time.Second))
+		client, err := client.NewEthereumPaymentClient(cfg, user, tcp.NewTCPBackend(5*time.Second))
 		assert.NoError(t, err)
+		err = client.Close()
+		assert.NoError(t, err)
```

Comment:
```suggestion
		assert.NoError(t, client.Close())
```

Reply:
Updated in cc40df19408154579443ce9af38e9aa743e8c9a2.